sites/all/modules/hr/hr_core/hr_core.module

```diff
@@ -1416,3 +1416,49 @@ function hr_core_entity_property_info_alter(&$entity_info) {
     'schema field' => 'changed',
   );
 }
+
+/**
+ * Implements hook_node_validate()
+ * Makes sure clusters, operations and offices have an English translation
+ */
+function hr_core_node_validate($node, $form, &$form_state) {
+  $node_types = array('hr_bundle', 'hr_operation', 'hr_office');
+  if (in_array($node->type, $node_types) && $node->language !== 'en') {
+    // Make sure the node has an English translation
+    $tnid = $node->tnid;
+    if ($tnid) {
+      $translations = translation_node_get_translations($tnid);
+      $languages = array_keys($translations);
+      if (!in_array('en', $languages)) {
+        form_set_error($form['language']['#name'], t('This node must have an English translation'));
```

Comment:
Is this error popping up in multiple contexts or could we make it say "this *list* must have an EN translation" ? I almost always try to avoid using the word "node" in a Drupal form error since it's such a generic term.

Reply:
It's popping up when trying to save a cluster, an operation or an office... Happy to change the wording, but list wouldn't be appropriate I think since here the user is not saving an HID list, but a cluster, operation or office...

Reply:
ah, now I see the `$node_types` array. In that case I don't have a better suggestion at the moment.

Reply:
I suppose you could provide a second array that says:

```php
$node_labels = ['Cluster', 'Operation', 'Office'];
```

Then substitute inside the `t()`, but it's a pretty minor issue overall.